nuxeo-features/nuxeo-elasticsearch/nuxeo-elasticsearch-automation/src/main/java/org/nuxeo/ecm/automation/elasticsearch/ElasticsearchWaitForIndexingOperation.java

```diff
@@ -66,24 +71,38 @@
     @Param(name = "refresh", required = false)
     protected Boolean refresh = false;
 
+    @Param(name = "waitForAudit", required = false)
+    protected Boolean waitForAudit = false;
+
     @OperationMethod
     public Boolean run() {
+        if (!isAdministrator()) {
```

Comment:
Usually, this is done with an XML contribution, as for `Elasticsearch.Index` in https://github.com/nuxeo/nuxeo/blob/master/nuxeo-features/nuxeo-elasticsearch/nuxeo-elasticsearch-automation/src/main/resources/OSGI-INF/operations-contrib.xml#L10

Reply:
+1, please use XML